### ISSUE TITLE
Hexagonal grid support?

### ISSUE THREAD

**Issue by eranimo** [englercj/gl-tiled#10]
I can't find any mention of hexagonal grid support. Does this support hexagon grids?

**Comment by englercj** [comment]
No, only orthographic square tile grids are currently supported.

I'm open to PRs though :)
